src/yyjson.c

```diff
@@ -7251,6 +7329,11 @@ char *yyjson_val_write_opts(const yyjson_val *val,
         return NULL;
     }
     
+#if YYJSON_DISABLE_NON_STANDARD
+    /*flg &= ~YYJSON_WRITE_ALLOW_INF_AND_NAN;*/
```

Comment:
@ibireme `YYJSON_DISABLE_NON_STANDARD` implies disabled `YYJSON_WRITE_ALLOW_INF_AND_NAN` but tests fail in this case. Will you handle it later?

I think the tests should check `YYJSON_DISABLE_NON_STANDARD` before testing JSON with `YYJSON_WRITE_ALLOW_INF_AND_NAN` flag.

Reply:
You're right, I'll fix it later.